src/main/cli/cli.c

```diff
@@ -280,7 +280,7 @@ static const char *mcuTypeNames[] = {
     "AT32F435"
 };
 
-static const char *configurationStates[] = { "UNCONFIGURED", "CUSTOM DEFAULTS", "CONFIGURED" };
+static const char *configurationStates[] = { "UNCONFIGURED", "CONFIGURED" };
```

Comment:
(you can use designate initializers to avoid enum problems
```suggestion
static const char *configurationStates[] = { 
 [CONFIGURATION_STATE_UNCONFIGURED ] = "UNCONFIGURED", 
 [CONFIGURATION_STATE_CONFIGURED ] = "CONFIGURED" };
```

Reply:
Thanks!